rxdogtag-autodispose/build.gradle

```diff
@@ -29,4 +30,15 @@ dependencies {
   testImplementation project(':rxdogtag').sourceSets.test.output
 }
 
+dokka {
+  externalDocumentationLink {
+    url = new URL("http://reactivex.io/RxJava/2.x/javadoc/")
+  }
+
+  outputFormat = 'gfm'
+  outputDirectory = "$rootDir/docs/0.x"
+  classpath = new ArrayList<File>(project.tasks['assemble'].outputs.files.files)
```

Comment:
This can just be the groovy `[]` initializer

Reply:
It wasn't able to cast it to a List<File>.